lib/puppet-lint/plugins/check_deprecated_functions/deprecated_functions.rb

```diff
@@ -0,0 +1,27 @@
+# Public: A puppet-lint custom check to detect deprecated functions.
+DEPRECATED_FUNCTIONS_VAR_TYPES = Set[:FUNCTION_NAME]
+
+# These functions have been deprecated in stblib and will impact puppet upgrades
```

Comment:
```suggestion
# These functions have been deprecated in stdlib and will impact puppet upgrades
```

Reply:
This one is definitely in the spirit of team effort, thanks @bastelfreak 👍